experimental/GaloisGrp/src/Subfields.jl

```diff
@@ -109,6 +110,7 @@ function Base.:*(A::SubfieldLatticeElem, B::SubfieldLatticeElem)
   cs = B.b
   ds = [intersect(b, c) for b = bs for c = cs]
   ds = [x for x = ds if length(x) > 0]
+  sort!(ds)
```

Comment:
Is this necessary, or for debugging, or what? (same for the other `sort!` calls you added)